### PR TITLE
[226] - [Markup] Create Nudge Component

### PR DESCRIPTION
**Pull request by johnpaul-sun**
## Issue Link
- https://app.asana.com/0/1203011167276287/1203196413501226/f

## Definition of Done
- [x] User can nudge other member

## Pre-condition
- yarn

## Expected Output
- when the user clicks the nudge button. a popup message should appear on the upper right corner.

## Screenshots/Recordings
image

### CODE REVIEW

File: client/src/utils/nudgeMember.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <button
+          onClick={() => toast.dismiss(t.id)}
+          className="w-full border border-transparent rounded-none rounded-r-lg p-4 flex items-center justify-center text-sm font-medium text-indigo-600 hover:text-indigo-500 focus:outline-none focus:ring-2 focus:ring-indigo-500"
+        >
+          Close
+        </button>
+      </div>
+    </div>
+  ))
+}
```
Comment (abduljalilpalala) [inline client/src/utils/nudgeMember.tsx]: Add new line

**Review by abduljalilpalala — COMMENTED** [review]
I have a minor comment, please check 🙇

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✔️

**Review by jsvelte — APPROVED** [review]
# Approved 💯

**Review by rogeliojohnoliverio — APPROVED** [review]
approved 🙇

**Pull request merged**